### DISCUSSION TITLE
i would like to discuss about the VRCX

### DISCUSSION THREAD

**Discussion by vKhalid** [vrcx-team/VRCX#191] [General]
**I have mini Idea's  of VRCX
 
I have a problem or by your update VRCX you not allowing to work when I press that Button  "Show avatar Details" when I press in any one whom I have into my Friend list actually it does not working  I hope you understand me what I side.**

-What is your pointe of being doing VRCX ?  what is your goal ??

**Comment by Myrkie** [comment]
refer to https://github.com/pypy-vrc/VRCX/issues/189
https://github.com/pypy-vrc/VRCX/issues/182

vrchat removed requesting of private and public avatars

**Reply by Myrkie** [reply]
will not be fixed/impossible to
